server/plugin/user.go

```diff
@@ -167,6 +167,9 @@ func (c *client) GetMe(mattermostUserID string) (*serializer.ServiceNowUser, err
 	if len(userDetails.UserDetails) == 0 {
 		return nil, fmt.Errorf("user doesn't exist on ServiceNow with email %s", mattermostUser.Email)
 	}
+	if len(userDetails.UserDetails) > 1 {
+		c.plugin.API.LogWarn("multiple users with the same email address on instance.", "Email", mattermostUser.Email)
```

Comment:
do we have anything for the instance as well like name aur URL we can also show that here with instance

Reply:
updated